--- gnustep-examples-0.9.7/gui/GSTest/main.m.orig	Wed Apr 24 17:34:06 2002
+++ gnustep-examples-0.9.7/gui/GSTest/main.m	Sun Dec 29 18:01:47 2002
@@ -51,8 +51,10 @@
 {
   [super init];
   // The test bundles are one directory up
-  bundlesPath = [[[[NSBundle mainBundle] bundlePath] 
-		   stringByDeletingLastPathComponent] retain];
+  bundlesPath = [[[[[[NSBundle mainBundle] bundlePath] 
+		   stringByDeletingLastPathComponent ] 
+		   stringByDeletingLastPathComponent ] 
+		   stringByAppendingString: @"/Library/Bundles" ] retain];
   return self;
 }
 
@@ -150,9 +152,8 @@
       
       // Determine which Bundle to load.  Bundles are as in 
       // bundlesPath/NSScrollView-test/NSScrollView-test.bundle
-      bundlePath = [[[bundlesPath stringByAppendingPathComponent: 
+      bundlePath = [[bundlesPath stringByAppendingPathComponent: 
 				    testList[i].bundleName] 
-		      stringByAppendingPathComponent: testList[i].bundleName]
 		     stringByAppendingPathExtension: @"bundle"];
       ASSIGN(testState[i].principalClassInstance,
 	     [self loadAndStartTestWithBundlePath: bundlePath]);
